test/tap/groups/pgsql-repl/bin/docker-pgsql-post.bash

```
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+. constants
+
+POSTGRE_SETUP_DIR=$(dirname $(realpath $0))/../scripts/
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)